Documentation/install/system_requirements.rst

```diff
@@ -149,3 +149,61 @@ and ``tc``, which is part of iproute2, to load BPF programs into the kernel.
 The minimum version of iproute2_ must be >= 4.8.0. Please see
 https://www.kernel.org/pub/linux/utils/net/iproute2/ for documentation on how
 to install iproute2.
+
+Firewall Rules
+========
+
+If you are running Cilium in an environment that requires firewall rules to enable connectivity, you will have to add the following rules to ensure Cilium works properly.
+
+At a minimum, all nodes running Cilium in a given cluster must be able to ping each other so ``cilium-health`` can report and monitor connectivity among nodes. This requires ICMP Type 0/8, Code 0 open among all nodes. TCP 4240 must also be open among all nodes for ``cilium-health`` monitoring.
```

Comment:
Awesome. Let's rephrase this a bit and state that it is recommended but optional. If the firewall does not permit it, CIlium will still operate fine but will not be able to provide health information. It is also an option to only allow one of the two methods.

Reply:
Agreed; change incoming